packages/apollo/src/commands/service/check.ts

```diff
@@ -69,3 +93,57 @@ export default class ServiceCheck extends ProjectCommand {
     return;
   }
 }
+
+// Validates flag variables and:
+//   1) returns an input object for the checkSchema query (HistoricQueryParameters)
+//   2) or throws an error
+export function validateHistoricParams({
```

Comment:
This probable deserves to live on its own outside of this service command. I think we should try to only have the command class when possible in these files (I know I added something else earlier on haha)